src/Model/ModelManagerInterface.php

```diff
@@ -338,4 +340,11 @@ public function getPaginationParameters(DatagridInterface $datagrid, $page);
      * @phpstan-param class-string $class
      */
     public function addIdentifiersToQuery($class, ProxyQueryInterface $query, array $idx);
+
+    // NEXT_MAJOR: Uncomment this method.
+//    /**
+//     * @phpstan-template T
```

Comment:
No need to add template if there is no IN/OUT or more than one use of the T param.